config/initializers/ab_tests.rb

```diff
@@ -18,4 +18,12 @@ module AbTests
         nil,
     }.compact,
   )
+
+  ACUANT_SDK = AbTestBucket.new(
+    experiment_name: 'Acuant SDK Upgrade',
+    buckets: {
+      use_newer_sdk: IdentityConfig.store.idv_acuant_sdk_upgrade_a_b_testing_enabled ?
+        IdentityConfig.store.idv_acuant_sdk_upgrade_a_b_testing_percent : nil,
+    }.compact
```

Comment:
my vote is to just have one feature flag, the percent, and we just scale that up or down (and make sure the AB test bucket code knows that `0` means "don't pick this ever"

```suggestion
    buckets: {
      use_newer_sdk: IdentityConfig.store.idv_acuant_sdk_upgrade_a_b_testing_percent,
    },
```

Reply:
I just confirmed via a spec locally that `0` is skipped, because we do a strictly > check